src/Symfony/Component/DependencyInjection/CHANGELOG.md

```diff
@@ -4,6 +4,7 @@ CHANGELOG
 3.4.0
 -----
 
+ * added `EnvProviderInterface` and corresponding "container.env_provider" tag for processing env vars
```

Comment:
I don't like the "provider" name here as implementations do not provide anything. They **process** env vars, they **filter** them (Twig's name). So, I would use something like `EnvProcessorInterface` and `container.env_processor`.

Reply:
To be even more precise, it's processors for env vars. So, could be `EnvVarProcessorInterface`.

Reply:
done